public/keymaps/primekb_prime_e_default.json

```
@@ -0,0 +1,12 @@
+{
+  "keyboard": "primeKB/prime_e",
```

It's best to treat this as if the entire interaction between qmk_configurator and qmk_firmware is case-sensitive. Future changes to the API may cause this setting to fail.

```suggestion
  "keyboard": "primekb/prime_e",
```